Formula/rubyfmt.rb

```diff
@@ -1,8 +1,10 @@
 class Rubyfmt < Formula
   desc "Ruby autoformatter"
   homepage "https://github.com/penelopezone/rubyfmt"
-  url "https://github.com/penelopezone/rubyfmt/archive/v0.2.0.tar.gz"
-  sha256 "68ebc0fd30933b1e27b609cc34c69a3bc886747d11c5b949e460ce01814adaeb"
+  url "https://github.com/penelopezone/rubyfmt.git",
+    branch:   "trunk",
+    tag:      "v0.8.0",
+    revision: "ed99cc4586a908c97f8b19ed78801342f7aa8512"
   license "MIT"
```

Comment:
```suggestion
  license "MIT"
  head "https://github.com/penelopezone/rubyfmt.git", branch: "trunk"
```

Reply:
You don't need both `branch` and `tag`, just `tag` for the stable download. I notice that you did lose the `head` portion though, hence my suggestion.